lib/honeybadger/agent.rb

```diff
@@ -413,6 +413,8 @@ def event(event_type, payload = {})
       extra_payload = {}.tap do |p|
         p[:request_id] = context_manager.get_request_id if context_manager.get_request_id
         p[:hostname] = config[:hostname].to_s if config[:'events.attach_hostname']
+        event_context = context_manager.get_event_context
+        p[:context] = event_context if event_context && !event_context.empty?
```

Comment:
The other implementations merge the context into the root

Reply:
Thanks for the heads-up!

Reply:
Fixed in https://github.com/honeybadger-io/honeybadger-ruby/pull/700/commits/0e48a0163b9bfb83979d97658527de2067206ec0